build.gradle.kts

```diff
@@ -26,3 +26,9 @@ task<Wrapper>("wrapper") {
     gradleVersion = SamplesVersions.gradleWrapper
     distributionType = Wrapper.DistributionType.ALL
 }
+
+tasks {
+    "ktlintCheck" {
+        dependsOn(":samples:kotlin-rulesets-creating:build")
```

Comment:
This is not required here, `kotlin-rulesets-using` already defined task dependency.

Reply:
Hmm, I thought that, but it does not work (CI link is here https://travis-ci.org/JLLeitschuh/ktlint-gradle/builds/370963993 ) , so I added the commit c8c0ead .
I'll try to do it again. :+1:

Reply:
After removing this:
```bash
$./gradlew -m :samples:kotlin-rulesets-using:ktlintCheck
:samples:kotlin-rulesets-creating:compileKotlin SKIPPED
:samples:kotlin-rulesets-creating:compileJava SKIPPED
:samples:kotlin-rulesets-creating:processResources SKIPPED
:samples:kotlin-rulesets-creating:classes SKIPPED
:samples:kotlin-rulesets-creating:jar SKIPPED
:samples:kotlin-rulesets-creating:assemble SKIPPED
:samples:kotlin-rulesets-creating:ktlintMainCheck SKIPPED
:samples:kotlin-rulesets-creating:ktlintTestCheck SKIPPED
:samples:kotlin-rulesets-creating:compileTestKotlin SKIPPED
:samples:kotlin-rulesets-creating:compileTestJava SKIPPED
:samples:kotlin-rulesets-creating:processTestResources SKIPPED
:samples:kotlin-rulesets-creating:testClasses SKIPPED
:samples:kotlin-rulesets-creating:test SKIPPED
:samples:kotlin-rulesets-creating:check SKIPPED
:samples:kotlin-rulesets-creating:build SKIPPED
:samples:kotlin-rulesets-using:ktlintMainCheck SKIPPED
:samples:kotlin-rulesets-using:ktlintTestCheck SKIPPED
:samples:kotlin-rulesets-using:ktlintCheck SKIPPED

BUILD SUCCESSFUL in 0s
```

Reply:
Yep, `./gradlew -m :samples:kotlin-rulesets-using:ktlintCheck` is successful in my own environment, too.

But `./gradlew ktlintCheck --no-daemon` with root project is failed.
If possible, Could you try to run this task?
This task is running on Travis CI with root project. (refs: https://github.com/JLLeitschuh/ktlint-gradle/blob/master/.travis.yml#L14)

The error log in my own environment is here: https://gist.github.com/uzzu/dfefe9d6341133af5fd3a6f58292b83c

Reply:
Forget that `ktlintCheck` is a meta task that is using to trigger tasks for different project source sets. Change `samples/kotlin-rulesets-using/build.gradle.kts` to use following:
```kotlin
afterEvaluate {
    tasks.findByName("ktlintMainCheck")?.dependsOn(":samples:kotlin-rulesets-creating:build")
    tasks.findByName("ktlintTestCheck")?.dependsOn(":samples:kotlin-rulesets-creating:build")
}
```

This should work.

Reply:
I see that and fixed e8e83e9